requirements_tools/upgrade_requirements.py

```diff
@@ -78,8 +78,13 @@ def installed(requirements_file):
         return expected_pinned
 
 
-def dirs(tmp):
-    dirnames = ('venv', 'venv/bin/python', 'venv/bin/pip')
+def dirs(tmp, pip_tool):
```

Comment:
hmm, this function probably changed meaning after I originally wrote it >.< not sure what a good name for it would be now :S